src/messages/kendo.messages.fr-CA.js

```diff
@@ -1,584 +1,1244 @@
-  "linkText": "Text",
-  "linkToolTip": "Info-bulle",
-  "linkWebAddress": "Adresse Web",
-  "search": "Search",
-  "createTable": "Insérer un tableau",
-  "addColumnLeft": "Add column on the left",
-  "addColumnRight": "Add column on the right",
-  "addRowAbove": "Add row above",
-  "addRowBelow": "Add row below",
-  "deleteColumn": "Supprimer la colonne",
-  "deleteRow": "Supprimer ligne",
-  "dropFilesHere": "drop files here to upload",
-  "formatting": "Format",
-  "viewHtml": "View HTML",
-  "dialogUpdate": "Update",
-  "insertFile": "Insert file",
-  "dialogOk": "OK",
-  "tableWizard": "Assistant de tableau",
-  "tableTab": "Table",
-  "cellTab": "Cellule",
-  "accessibilityTab": "Accessibilité",
-  "caption": "Sous-titre",
-  "summary": "Sommaire",
-  "width": "Largeur",
-  "height": "Hauteur",
-  "cellSpacing": "Espacement des cellules",
-  "cellPadding": "Rembourrage des cellules",
-  "cellMargin": "Marge des cellules",
-  "alignment": "Alignement",
-  "background": "Fond",
-  "cssClass": "CSS Classe",
-  "id": "Id",
-  "border": "Bordure",
-  "borderStyle": "Style de bordure",
-  "collapseBorders": "Rétracter bordures",
-  "wrapText": "Renvoi à la ligne",
-  "associateCellsWithHeaders": "Entêtes associées",
-  "alignLeft": "Aligner à gauche",
-  "alignCenter": "Aligner le centre",
-  "alignRight": "Aligner à droite",
-  "alignLeftTop": "Aligner à gauche et haut",
-  "alignCenterTop": "Aligner le centre et haut",
-  "alignRightTop": "Aligner à droite et haut",
-  "alignLeftMiddle": "Aligner à gauche et milieu",
-  "alignCenterMiddle": "Aligner le centre et milieu",
-  "alignRightMiddle": "Aligner à droite et milieu",
-  "alignLeftBottom": "Aligner à gauche et bas",
-  "alignCenterBottom": "Aligner le centre et bas",
-  "alignRightBottom": "Aligner à droite et bas",
-  "alignRemove": "Retirer alignement",
-  "columns": "Colonnes",
-  "rows": "Lignes",
-  "selectAllCells": "Sélectionner toutes les cellules"
-});
-}
-
-/* FileBrowser and ImageBrowser messages */
-
-var browserMessages = {
-  "uploadFile" : "Charger",
-  "orderBy" : "Trier par",
-  "orderByName" : "Nom",
-  "orderBySize" : "Taille",
-  "directoryNotFound" : "Aucun répértoire de ce nom.",
-  "emptyFolder" : "Répertoire vide",
-  "deleteFile" : 'Etes-vous sûr de vouloir supprimer "{0}"?',
-  "invalidFileType" : "Le fichier sélectionné \"{0}\" n'est pas valide. Les type fichiers supportés sont {1}.",
-  "overwriteFile" : "Un fichier du nom \"{0}\" existe déjà dans ce répertoire. Voulez-vous le remplacer?",
-  "dropFilesHere" : "glissez les fichiers ici pour les charger",
-  "search" : "Recherche"
-};
-
-if (kendo.ui.FileBrowser) {
-kendo.ui.FileBrowser.prototype.options.messages =
-$.extend(true, kendo.ui.FileBrowser.prototype.options.messages, browserMessages);
-}
-
-if (kendo.ui.ImageBrowser) {
-kendo.ui.ImageBrowser.prototype.options.messages =
-$.extend(true, kendo.ui.ImageBrowser.prototype.options.messages, browserMessages);
-}
```

Comment:
ImageBrowser translations are removed, only FileBrowser translations remain after the update